multitouch.h

```diff
@@ -30,6 +30,14 @@
 # include <linux/input.h>
 # include <stdbool.h>
 
+#ifndef input_event_sec
+#define input_event_sec time.tv_sec
```

Comment:
Why not define these two macros in musl-compatibility and include that header from multitouch.h and event-input.h ?

Reply:
There was a short discussion on this on the upstream SailfishOS side as well. https://git.sailfishos.org/mer-core/mce/merge_requests/148#note_83843

Note that this define isn't actually needed for Musl but only for older kernels which aren't patched with time64 yet on 32-bit arches. `musl-compatibility.h` would be the wrong place, as this on itself doesn't really have anything to do with Musl.